src/features/MovieList/popularMoviesSlice.js

```diff
@@ -0,0 +1,35 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+const popularMoviesSlice = createSlice({
+    name: "popularMovies",
+    initialState: {
+        status: "initial",
+        popularPeopleData: null,
+    },
+    reducers: {
+        fetchPopularMoviesLoading: () => ({
```

Comment:
I would simply call it `fetchPopularMovies`. The same way Chris showed it in ToDoList.

Reply:
please leave it for now.  as this reducer only sets status to loading, it should has loading in name.